wpiutil/src/main/native/include/wpi/jni_util.h

```diff
@@ -593,6 +593,81 @@ class JSingletonCallbackManager : public JCallbackManager<T> {
   }
 };
 
+inline std::string GetJavaStackTrace(JNIEnv* env, StringRef skipPrefix) {
```

Comment:
It would not surprise me if this is less efficient than doing the same operation in Java and passing down a string to JNI instead.  But I guess the advantage here is you can get it directly from the JNI side instead of having to pass a string.

Reply:
Yeah I'd probably prefer to do this on native end. I'm not super worried about efficiency here, its not an every call thing, and its not _that_ much slower.